### PR TITLE
New NPM publishing docs

### PR DESCRIPTION
**Pull request by tphalp**
Added NPM.md to outline how publishing to NPM works.

### CODE REVIEW

File: ADAPTERS.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 Each aggregator that wants to participate in the UCP needs to create their own adapter package. This allows the UCW to interact with the aggregator.
 
-> ⚠️ This repository only contains test adapters to allow the UCP to have test paths for aggregator adapters. Real aggregator adapters should be created in a fork of this repository. See [the MX Adapter Package Fork](https://github.com/Universal-Connect-Project/ucw-adapter-mx) for an example of how an adapter package can be created. 
+> [!NOTE]  
+> This repository only contains test adapters to allow the UCP to have test paths for aggregator adapters. Real aggregator adapters should be created in a fork of this repository. See [the MX Adapter Package Fork](https://github.com/Universal-Connect-Project/ucw-adapter-mx) for an example of how an adapter package can be created.
```
Comment (tphalp) [inline ADAPTERS.md:8]: This is a new Markdown format that allows "alerts". It is a nice way to format things that should pop out to the user so they don't ignore it.

File: NPM.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+In order for end-users to use your adapter with the UCW, you need to publish it to NPM. End-users can then install your adapter from NPM, and then set it up with their own instance of the UCW.
+
+## How to publish to NPM?
+
+We've tried to make this process as simple as possible. 
+
+Publishing your adapter package to NPM is a mostly automated process that happens as part of the GitHub Actions workflows that run during the PR process. The workflow for publishing to NPM happens when a PR is merged to `main`.
+
+In order to publish your adapter package to NPM as part of the workflow, you'll need to do the following.
+f
```
Comment (emilysallstrom) [inline NPM.md:20]: Is this `f` intentional?

**Review by emilysallstrom — DISMISSED** [review]

**Comment by wesrisenmay-mx** [comment]
Nice work. 1 issue relayed verbally.

File: NPM.md (hunk 2 above)
Comment (tphalp) [reply NPM.md]: Nope! Thanks. Fixed.

**Review by wesrisenmay-mx — APPROVED** [review]

**Pull request merged**